docker-compose.yml

```diff
@@ -36,7 +36,7 @@ services:
   python:
     build:
       context: ./src
-      dockerfile: Dockerfile.prod
+      dockerfile: prod.Dockerfile
```

Comment:
Waarom hernoem je dit maar niet de frontend dockerfile?

Reply:
Ik dacht ik verander alleen wat ik aanraak. Maar bij deze allebeid aangepast. Ik vind `prod.Dockerfile` wel handiger dan `Dockerfile.prod` omdat de IDE het dan herkent als Dockefile. 

Nadeel is dat bv `dev.Dockerfile` en `prod.Dockerfile` niet meer vlak onder elkaar staan, maar als we dat echt willen kunnen we ze in een los mapje stoppen.

Reply:
Prima oplossing!